src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx

```diff
@@ -101,7 +101,7 @@ Similarly `clamp_min(duration, 1)` will not return any value lower than 1. The f
 
 ### `pow`
 
-`pow(n, m)` computes _n_ raised to the power _m_. (I.e. `n * n * ... * n,` with _m_ copies of _n_)
+`pow(n, m)` computes _n_ raised to the power _m_. (for example, `n * n * ... * n,` with _m_ copies of _n_)
```

Comment:
@austin-schaefer, this one looks good. On line 72, there's a "For example" that could use a following comma.

Reply:
For me, the line 72 one reads better without a comma?